<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_modifier</class>
 <widget class="QDialog" name="login_modifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load_table">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>load table</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_etat">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_text_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Identifiant</string>
   </property>
   <widget class="QLineEdit" name="text_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="text_autorisation">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_text_id_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mot de passe</string>
   </property>
   <widget class="QLineEdit" name="text_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="text_temp_id">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="radio_group_etat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Etat</string>
   </property>
   <widget class="QRadioButton" name="radio_actif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actif</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_suspendu">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Suspendu</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="radio_group_autorisation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Autorisation</string>
   </property>
   <widget class="QRadioButton" name="radio_admin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Administrateur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_docteur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Docteur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_secretaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Secr√©taire</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>0</y>
     <width>791</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/Background_color.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_valider">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/button_init.png);</string>
   </property>
   <property name="text">
    <string>   Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_add_login_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/checked.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="icon_add_login_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/Gestion login/rubbish.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_supprimer">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/button_init.png);</string>
   </property>
   <property name="text">
    <string>    Supprimer</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/btn_gestion_patient1.png)3 10 15 10;
font: 18pt &quot;Bahnschrift&quot;;
</string>
   </property>
   <property name="text">
    <string>Modifier un Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_edit_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/Gestion login/icon_edit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>text_temp_id</zorder>
  <zorder>btn_load_table</zorder>
  <zorder>text_autorisation</zorder>
  <zorder>text_etat</zorder>
  <zorder>label</zorder>
  <zorder>tableView</zorder>
  <zorder>group_text_id</zorder>
  <zorder>group_text_id_2</zorder>
  <zorder>radio_group_etat</zorder>
  <zorder>radio_group_autorisation</zorder>
  <zorder>btn_valider</zorder>
  <zorder>icon_add_login_2</zorder>
  <zorder>btn_supprimer</zorder>
  <zorder>icon_add_login_3</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>btn_edit_login</zorder>
  <zorder>icon_edit_login</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
